share/instruments/instruments.xml

```diff
@@ -7489,9 +7489,9 @@
             </Instrument>
             <Instrument id="bells">
                   <family>unpitched-metal-percussion</family>
-                  <trackName>Bells</trackName>
-                  <longName>Bells</longName>
-                  <shortName>Be.</shortName>
+                  <trackName>Ride Bell</trackName>
+                  <longName>Ride Bell</longName>
+                  <shortName>Ri. Be.</shortName>
```

Comment:
This change (at least) requires a run of generateTs.py to get the translatable strings updated into instrumetnxml.h

Reply:
`generateTs.py` is not currently working 100% but will be fixed before release. See https://github.com/musescore/MuseScore/pull/8835#discussion_r691019178